camunda-formio-tasklist-vue/src/components/TaskList.vue

```diff
@@ -397,25 +386,25 @@ const StoreServiceFlowModule = namespace("serviceFlowModule");
 @Component({
   components: {
     DatePicker,
-    TaskHistory,
-    Header,
-    LeftSider,
+    TaskHistory: () => import ("../components/addons/TaskHistory.vue"),
```

Comment:
what is the advantage of this?

Reply:
This is for LazyLoading of components

Reply:
It's for lazy loading. We haven't full taken advantage of it yet.